docs/tutorials/google/compare_to_calibration.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<p>"For more background on calibration API.." can move up to the cell where we are defining the metric  and plotting the heatmap via the calibration object. </p><p> </p><p>Let's add a small description about computing the average error for every selected set of qubits.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Moved links to tutorial description + relevant sections. Computing average is no longer relevant as the echo on different segments was removed.</p>